.larasail/database

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+#/
+#--------------------------------------------------------------------------
+# Larasail Database Manager
+#--------------------------------------------------------------------------
+#
+# This command gives the ability to create & manage database
+# for a Larasail project.
+#
+# Default database user and password are: 'larasail'.
+# You are free to change them with --user & --db flags.
+#
+# Upon successful database creation this command also tries
+# to update .env in the current directory if it is present.
+#
+#/
+
+. /etc/.larasail/includes/colors
+. /etc/.larasail/includes/format
+
+FORCE_INIT=false
+
+DATABASE_USER=larasail
+DATABASE_NAME=larasail
```

Comment:
Great job on all that! Looks really good 🙌

Just one remark, I think that it would be good to maybe ask the user to specify the database name (and set the database username to match the database name) as the user might want to run that multiple times and have multiple databases in case that they have more than 1 website on the server.

Reply:
Currently user is able to specify `--user` and `--db` flags in order to override those values (which are set to `larasail` by default.

We can either try and improve visibility of those options (right now you can get help message by typing `larasail database`) or change flags into an interactive prompt).

Both approaches have they strengths and weaknesses (in regards to interactivity), so I think it's worth doing both: leave those flags but ask with a prompt if flags are not specified. This way user don't have to guess.

I will update the PR.